manifests/configs/nginx.conf

```diff
@@ -22,6 +22,6 @@ server {
 
   # Configured in settings.py with Django
   location /assets/ {
-    root /usr/share/nginx/html;
+    root /usr/share/nginx;
```

Comment:
note: client access like `http://example.com/assets/bennuhp/js/main.js` is matched this pattern, and redirected onto `/usr/share/nginx/assets/bennuhp/js/main/js`.
(Meaning, root path + matched URL pattern: `/usr/share/nginx` + `/assets/bennuhp/js/main/js`)